README.md

```diff
@@ -1 +1,102 @@
-# api-Webapp
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# API-Webapp
+
+This project is about an exhibition app used to display, comment on and reserve art. The page was built using JavaScript and bundled with Webpack and uses API calls. The webapp has 3 user interfaces:
+  - A home page showing a list of items that you can "like".
+    * Made by: [Molnar Emese](https://github.com/Mesi21)
+  - A popup window with more data about an item that you can use to leave a comment.
+    * Made by: [Jeff Kayombo](https://github.com/JEFFKAY16)
+  - A popup window with more data about an item that you can use to make a reservation.
+    * Made by: [Hamza Tarar](https://github.com/Lockless404)
+
+![Homepage Screenshot](./webApiHome.png)
+
+![Reservation Popup Screenshot 1](./webApiRes1.png)
+![Reservation Popup Screenshot 2](./webApiRes2.png)
+
+![Comment Popup Screenshot 1](./webApiCom1.png)
+![Comment Popup Screenshot 2](./webApiCom2.png)
+
+## Built With
+
+* JavaScript
+* Webpack
+* HTML
+* CSS
+
+## Tested With
+
+* Jest
+
+## Aditional Resources
+
+* Kanban Board
+* Involvment API
+* [Lorem Picsum](https://picsum.photos) 
+* Linters: Webhint, Lighthouse, StyleLint, ESLint
+
+## Video description of project
+
+[Watch video here](https://drive.google.com/file/d/10Df8piuumCMcRFjah8a-kHuLXsJZMg8w/view?usp=sharing)
+
+## Live Demo
+
+[Live Demo](https://cocky-shockley-f14242.netlify.app/)
+
+
+## Getting Started
+
+To get a local copy up and running follow the next steps:
+> Clone the repo to you local machine.
+```bash
+git clone "https://github.com/Lockless404/api-Webapp.git"
+run `npm install`
+run `npm start` to start the server
+```
```

Comment:
[OPTIONAL]

- Everything looks good. As this is a small change I'm making it optional but please include this command.Please mention the command on how to test the application. [Hint. `npm test`]